contrib/dracut/90zfs/zfs-load-key.sh.in

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# This script only gets executed on systemd systems, see mount-zfs.sh for non-systemd systems
+
+# import the libs now that we know the pool imported
+[ -f /lib/dracut-lib.sh ] && dracutlib=/lib/dracut-lib.sh
+[ -f /usr/lib/dracut/modules.d/99base/dracut-lib.sh ] && dracutlib=/usr/lib/dracut/modules.d/99base/dracut-lib.sh
+. "$dracutlib"
```

Comment:
Shouldn't we be using the `dracutdir` specified by `configure --with-dracutdir=DIR` here?

Reply:
I coppied that from zfs-generator.sh.in

Reply:
I think it needs to look in both locations because it's installed to /lib in the initram.  I think...